docs/migrating_to_nextcord.rst

```diff
@@ -8,3 +8,44 @@ Migrating to nextcord
 Due to the `original discord.py repository <https://github.com/Rapptz/discord.py>`_ becoming read-only, we decided
 that it would be necessary to fork it and keep on developing further. We also wanted to change the name and voted on
 nextcord in order to properly register it at pypi.
+
+Porting from discord.py
+-----------------------
+
+In order to port a bot using discord.py to nextcord.py, follow these steps:
+
+1. Install nextcord: 
+.. code:: sh
+
+    # Linux/macOS
+    python3 -m pip install -U nextcord
+
+    # Windows
+    py -3 -m pip install -U nextcord
+
+2. Update the following import statements:
+
+* ``import discord`` -> ``import nextcord as discord``
+* ``from discord.ext`` -> ``from nextcord.ext``
+
+Code Example
+------------
+
+.. code:: py
+    #Previously: import discord

```

Comment:
iirc this needs a blank space after directives

Reply:
Not sure what you mean by this, I believe you have to add an extra blank line for code to show but I did that in both of my examples. The format looks ok when i test it on a [rst editor](http://rst.ninjs.org/?theme=basic#TWlncmF0aW5nIHRvIG5leHRjb3JkCj09PT09PT09PT09PT09PT09PT09PT0KCkR1ZSB0byB0aGUgYG9yaWdpbmFsIGRpc2NvcmQucHkgcmVwb3NpdG9yeSA8aHR0cHM6Ly9naXRodWIuY29tL1JhcHB0ei9kaXNjb3JkLnB5PmBfIGJlY29taW5nIHJlYWQtb25seSwgd2UgZGVjaWRlZAp0aGF0IGl0IHdvdWxkIGJlIG5lY2Vzc2FyeSB0byBmb3JrIGl0IGFuZCBrZWVwIG9uIGRldmVsb3BpbmcgZnVydGhlci4gV2UgYWxzbyB3YW50ZWQgdG8gY2hhbmdlIHRoZSBuYW1lIGFuZCB2b3RlZCBvbgpuZXh0Y29yZCBpbiBvcmRlciB0byBwcm9wZXJseSByZWdpc3RlciBpdCBhdCBweXBpLgoKUG9ydGluZyBmcm9tIGRpc2NvcmQucHkKLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLQoKSW4gb3JkZXIgdG8gcG9ydCBhIGJvdCB1c2luZyBkaXNjb3JkLnB5IHRvIG5leHRjb3JkLnB5LCBmb2xsb3cgdGhlc2Ugc3RlcHM6CgoxLiBJbnN0YWxsIG5leHRjb3JkOiAKCi4uIGNvZGU6OiBzaAoKICAgICMgTGludXgvbWFjT1MKICAgIHB5dGhvbjMgLW0gcGlwIGluc3RhbGwgLVUgbmV4dGNvcmQKCiAgICAjIFdpbmRvd3MKICAgIHB5IC0zIC1tIHBpcCBpbnN0YWxsIC1VIG5leHRjb3JkCgoyLiBVcGRhdGUgdGhlIGZvbGxvd2luZyBpbXBvcnQgc3RhdGVtZW50czoKCiogYGBpbXBvcnQgZGlzY29yZGBgIC0+IGBgaW1wb3J0IG5leHRjb3JkIGFzIGRpc2NvcmRgYAoqIGBgZnJvbSBkaXNjb3JkLmV4dGBgIC0+IGBgZnJvbSBuZXh0Y29yZC5leHRgYAoKQ29kZSBFeGFtcGxlCi0tLS0tLS0tLS0tLS0tCgouLiBjb2RlOjogcHkKICAgICNQcmV2aW91c2x5OiBpbXBvcnQgZGlzY29yZAogICAgZnJvbSBuZXh0Y29yZCBpbXBvcnQgZGlzY29yZAoKICAgICNQcmV2aW91c2x5OiBmcm9tIGRpc2NvcmQuZXh0IGltcG9ydCBjb21tYW5kcwogICAgZnJvbSBuZXh0Y29yZC5leHQgaW1wb3J0IGNvbW1hbmRzCgogICAgYm90ID0gY29tbWFuZHMuQm90KGNvbW1hbmRfcHJlZml4PSckJykKCiAgICBAYm90LmNvbW1hbmQoKQogICAgYXN5bmMgZGVmIHBpbmcoY3R4KToKICAgICAgICBlbWJlZD1kaXNjb3JkLkVtYmVkKAogICAgICAgICAgICB0aXRsZT0iUG9uZyEiLAogICAgICAgICAgICBjb2xvdXI9ZGlzY29yZC5Db2xvci5ncmVlbigpCiAgICAgICAgKQogICAgICAgIGF3YWl0IGN0eC5zZW5kKGVtYmVkPWVtYmVkKQogICAgCiAgICBib3QucnVuKCdbQm90IFRva2VuXScp).

Reply:
Oh NVM, I figured out what you meant. Was looking in the wrong area. Thanks!

Reply:
https://i.imgur.com/eIKBGen.png after the py code-block directive it needs a blank like just like the shell (sh) code-block

Reply:
Added, thank you 🙂